roofit/histfactory/src/HistoToWorkspaceFactoryFast.cxx

```diff
@@ -2026,7 +2026,6 @@ namespace HistFactory{
     // clean up 
     delete model;
```

Comment:
Given the nice fix for the asimov_dataset perhaps we can use unique_ptr also for the other pointers?

Reply:
Absolutely! However, I didn't want to touch stuff not related to the bug report. Maybe I should go all-in at least for every function I touch.